queries.sql

```diff
@@ -95,14 +95,14 @@ GROUP BY species;
 
 /* query using JOIN */
 
-/* Join owner_full_name using the owners_id */
+/* What animals belong to Melody Pond? */
 
 SELECT animals.name
   FROM animals
 	JOIN owners ON animals.owner_id = owner_id
 	WHERE owners.full_name = 'Melody Pond';
 
-	/* List animals that are pokemon */
+/* List of all animals that are pokemon (their type is Pokemon). */
 
 SELECT animals.name
```

Comment:
- [ ] Like the previous CR mentioned you should use SELECT animals instead of SELECT animals.name according to the requirements

Reply:
Thank you for the explanation @mikethreels